components/motor/gpio/motor_encoder.go

```diff
@@ -316,8 +311,15 @@ func (m *EncodedMotor) rpmMonitor() {
 	lastTime := time.Now().UnixNano()
 
 	rpmSleep, rpmDebug := getRPMSleepDebug()
+	inRamp := false
+
 	for {
-		timer := time.NewTimer(rpmSleep)
+		myRpmSleep := rpmSleep
+		if inRamp {
+			// if we're ramping up or down, make the loop faster
+			myRpmSleep /= 4
```

Comment:
add a todo to finesse pls

Reply:
I generally don't like TODOs unless they are super specific.